site-root/cbl/powertools/reset-baselines.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+use Slate\CBL\Competency;
+use Slate\CBL\ContentArea;
+use Slate\CBL\StudentCompetency;
+
+use Slate\People\Student;
+
+
+$GLOBALS['Session']->requireAccountLevel('Developer');
+
+$students = [];
+if (isset($_REQUEST['students'])) {
+    $students = Student::getAllByListIdentifier($_REQUEST['students']);
+    $studentIds = array_map(function($s){
+        return $s->ID;
+    }, $students);
+}
+
+$contentArea = null;
+if (isset($_REQUEST['content_area'])) {
```

Comment:
does only using isset instead of !empty work when one of the filters is submitted with no value?